content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/fifo.rst

```diff
@@ -0,0 +1,98 @@
+============
+FIFO removal
+============
+
+The *First In, First Out* (FIFO) removal strategy selects products with the earliest arrival dates.
+This method is useful for companies selling products that have short demand cycles, like clothes,
+for example. By using :abbr:`FIFO (First In, First Out)`, companies can prevent prolonged stock
+retention of specific styles.
+
+.. example::
+   Various quantities of the product, `T-shirt`, tracked by lot numbers, arrive on August 1st and
+   August 25th. For an order made on September 1st, the :abbr:`FIFO (First In, First Out)` removal
+   strategy prioritizes lots that have been in stock the longest. So, products received on August
+   1st are selected first for picking.
```

Comment:
In the FEFO doc, you put the example after the Removal date section. Here, you have the example before the Arrival date section. I would make the order consistent throughout all of the docs.